meta/main.yml

```diff
@@ -8,8 +8,13 @@ galaxy_info:
   platforms:
     - name: EL
       versions:
+        - 6
```

Comment:
The role is fairly generic so it should work on `all` CentOS/RHEL distros. Please could you set the list of supported EL versions back to `- all`?